event-publisher-spi/scalastyle-config.xml

```diff
@@ -0,0 +1,93 @@
+<scalastyle>
```

Comment:
Not something you need to solve now, but should we have a shared scalastyle-config that all projects can use?

Reply:
Yeah. It would also be good to try and use a single sbt build file to create the separate SPI jars.